dataactcore/utils/jobQueue.py

```diff
@@ -34,7 +34,7 @@ def enqueue(jobID):
             response = requests.post(url, params)
             return response.json()
 
-        @self.jobQueue.task(name='jobQueue.generate_d1')
+        @self.jobQueue.task(name='jobQueue.generate_d_file')
```

Comment:
This is just the task name that's published when it runs. Below, it actually uses the correct task name to create the method reference. The method reference below is what's actually used but I wanted to update the task name to be consistent with the method name. Shouldn't have any affect otherwise